tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/report.json

```diff
@@ -122,94 +114,6 @@
         }]
       }]
     }]
-  }, {
```

Comment:
The current changes affect these statcheck counters that were designed to validate these counters.  Could you please use a single observationPeriod for all existing rows?  If you want to multiple periods, add new rows.

Reply:
fixed, please see:
- this summary_report [(raw html)](https://github.com/enjoythecode/import/blob/fix103/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) [(rendered html)](https://htmlpreview.github.io/?https://github.com/enjoythecode/import/blob/fix103/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) at `geoId/01`